lib/libbackscrub.cc

```diff
@@ -0,0 +1,270 @@
+/* This is licenced software, @see LICENSE file.
+ * Authors - @see AUTHORS file.
+==============================================================================*/
+
+// tested against tensorflow lite v2.4.1 (static library)
```

Comment:
Prone to be outdated. Especially in light of #84 …

Reply:
Another import from the original source, agreed not a helpful comment.